.gitignore

```diff
@@ -37,3 +37,4 @@ Thumbs.db
 luacov*
 *.out
 t/servroot
+t/redis-cluster-compose-v702.yaml
```

Comment:
Debug code?

Reply:
It is to launch the external dependency: Redis cluster.

Reply:
Remove it.